src/Client.php

```diff
@@ -139,19 +137,16 @@ private static function parseJwk($jwk)
 
         $encoded = base64_encode($pkey->getBinary());
 
-        return new \Lcobucci\JWT\Signer\Key(
+        return
```

Comment:
Kind of surprising that is not needed anymore. Why was it there in the first place?

Reply:
I guess I just wanted to fail earlier before, if the key was incorrect. But there's a `string` typehint on the `key` arg of `Token::verify`, so phpstan complained. The JWT lib constructs its own `Key` later on in `Token::verify`, and it doesn't really matter where we fail, so the change is harmless.